### PR TITLE
Update Slack : For attachments

### PR DESCRIPTION
**Pull request by mithunbharadwaj**

### CODE REVIEW

**Comment by mithunbharadwaj** [comment]
@ateska Slack with a pdf attachment

Screenshot 2023-04-06 at 10 58 41

File: asabiris/output/slack/service.py [hunk 1]
```diff
@@ -1,38 +1,54 @@
...
+
+        if self.SlackWebhookUrl is None:
+            return
+
+        try:
+            response = self.Client.chat_postMessage(
+                channel=self.Channel,
+                text=body
+            )
+            print("Message sent: ", response["ts"])
```
Comment (ateska) [inline asabiris/output/slack/service.py:40]: `print` should not be here.

**Review by ateska — CHANGES_REQUESTED** [review]

File: asabiris/output/slack/service.py [hunk 2]
```diff
@@ -1,38 +1,65 @@
...
+
+        try:
+            if len(atts) != 0:
+                for attachment in atts:
+                    file_content = attachment[0].encode('utf-8')
+                    file_obj = BytesIO(file_content)
+                    self.Client.files_upload(
+                        channels=self.Channel,
+                        file=file_obj,
+                        initial_comment=body,
```
Comment (mithunbharadwaj) [inline asabiris/output/slack/service.py:56]: @ateska Is this what you meant by 'Attachments should be part of the Slack message ... not uploaded separately.'?

Comment (ateska) [reply asabiris/output/slack/service.py:56]: In the single slack message

https://api.slack.com/messaging/webhooks#advanced_message_formatting

See "blocks"

**Review by ateska — CHANGES_REQUESTED** [review]

**Review by ateska — APPROVED** [review]

**Pull request merged**